### ISSUE TITLE
UI Test: Bank Name

### ISSUE THREAD

**Issue by murog** [GoogleCloudPlatform/bank-of-anthos#237]
### Description
Add UI test to ensure bank name is what is expected after #235 

### Expected Behavior
If BANK_NAME is not set, then application should default to "Bank of Anthos"

**Comment by bourgeoisor** [comment]
Closing in favour of https://github.com/GoogleCloudPlatform/bank-of-anthos/issues/278
